packages/react/src/components/CTA/CTA.js

```diff
@@ -1,127 +0,0 @@
-/**
```

Comment:
@kennylam this shouldn't have been deleted. This is a fundamental component used in the `ContentBlock`, as well as `ContentGroup` and `ContentItem`.

Reply:
@jeffchew Yeah this came in during a merge and I had no idea what it was/did. Incorrectly assumed it was supposed to be removed and blindly accepted change. It's restored